bundler.gemspec

```diff
@@ -47,6 +47,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "ronn",       "~> 0.7.3"
   s.add_development_dependency "rspec",      "~> 3.6"
   s.add_development_dependency "rubocop",    "= 0.50.0"
+  s.add_development_dependency "rspec_junit_formatter", "~> 0.2.3"
```

Comment:
I'm not sure about adding a new dev dependency. Does azure require this to work?

Reply:
Not really.

This enables the use of `-r rspec_junit_formatter --format RspecJunitFormatter -o rspec/bundler-junit-results.xml` as a param for `ruby bin/rspec` which creates this file, which can then be used by the `PublishTestResults@2` task to generate the test result analysis at https://dev.azure.com/janpio/bundler/_build/results?buildId=606&view=ms.vss-test-web.build-test-results-tab. That would be very handy to follow the improvements I make when the Windows testing bugfixes get merged, but of course one could also manually track the fixed tests. This is just a convenience thing.

I thought that it would worth that convenience as it is just a development dependency. But if it is easier to merge this PR without this added dependency, I can of course remove it and the part for the test result publishing from the CI config.

Reply:
I would prefer that you install `rspec_junit_formatter` as part of the setup steps in Azure Pipelines, since Bundler does not itself depend on the gem for development.

Reply:
Umm, I feel a bit dumb now, but: How does one do that? Just `gem install rspec_junit_formatter`?

Reply:
Hm, `ruby bin/rake spec:deps` would probbaly be one way, because that basically runs `gem install ...`. But that would also install it on travis, which is probably not wanted.

Reply:
Yeah, I am out of my depth here - I made a few commits but always get `C:/hostedtoolcache/windows/Ruby/2.4.3/x64/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- rspec_junit_formatter (LoadError)`. All help appreciated.

Reply:
Now is installed in an additional step in the Azure Pipelines configuration, so fully independent of the general codebase.